core/src/main/java/io/questdb/cairo/SymbolMapWriter.java

```diff
@@ -29,6 +29,7 @@
 import io.questdb.log.Log;
 import io.questdb.log.LogFactory;
 import io.questdb.std.*;
+import io.questdb.std.str.DirectCharSink;
```

Comment:
i think ```SingleCharCharSequence``` might work better here

Reply:
:) that's exactly what I was looking for